openpype/tools/ayon_launcher/ui/actions_widget.py

```diff
@@ -101,6 +101,8 @@ def refresh(self):
 
         groups_by_id = {}
         for action_items in items_by_label.values():
+            action_items.sort(key=lambda x: x.variant_label)
```

Comment:
Would doing `action_items.sort(key=lambda x: x.variant_label, reverse=True)` work? And skip the following line

Reply:
That'd likely be:
```python
action_items.sort(key=(lambda x: x.variant_label), reverse=True)
```

Also note that pep recommends to `def` the function instead of using `lambda` - right? (or is that purely for lambda assignments?):
```python
def sort_by_variant_label(item):
    return item.variant_label

actions_items.sort(key=sort_by_variant_label, reverse=True)
```

This could also be:
```python
action_items.sort(key=operator.attrgetter("variant_label"), reverse=True)
```
See [`operator.attrgetter`](https://docs.python.org/3/library/operator.html#operator.attrgetter)

Reply:
Guys, what about suggestions?

Reply:
> Guys, what about suggestions?

I wanted to comment in the thread - since it was on the single line originally I couldn't offer the multi-line suggestion needed to remove the line after it. Or does Github allow a way to do that still within this comment thread?

Plus - not sure which of these you'd have preferred yourself.